content-services/latest/install/ansible.md

```diff
@@ -313,6 +352,48 @@ For details about the webapp URLs, location of logs, configuration etc., see [us
 
 If you're deploying a production system, ensure that you review the additional information provided in [Securing your installation]({% link content-services/latest/admin/securing-install.md %}).
 
+### Additionnal command switches for ansible-playbook
+
+There are some useful argument you can use with `ansible-playbook` command in many circumstances. Some are highlighted bellow but take a look at [The ansible-playbook documentation](https://docs.ansible.com/ansible/latest/cli/ansible-playbook.html) for complete list of options.
+
+* `-k` : Prompt for SSH password. Usefull when no SSH keys have been deployed but needs to be th same on all hosts (prefer SSH whenever possible)
+* `-K` : Prompt for sudo password. Usefull when the user used to connect to the machine is not root
+* `-e` : Pass an extra variable or override an existing one.
+* `-l` : Limit the play to a subset of hosts (either groups or individuals hosts or a mix of both)
+* `-u user` : specify the username to use to cnnect to all targets (Prefer adding the ànsible_ssh_user` to the inventory file in the right scope, e.g. under the `all`group)
+
+## ACS cluster 
+
+Due to load or high availability needs, you might want to deploy a cluster of several repository nodes. This can be achieved rather simply by:
+
+* Giving the playbook the location of the shared storage used for the ACS contentstore (See [Shared storage documentation](shared-contentstore.md) for details).
+* Specifying several hosts within the repository hosts group
+
+> :warning: as mention in the [Alfresco official documentation](https://docs.alfresco.com/content-services/latest/admin/cluster/#scenarioredundancycluster), "All the servers in a cluster should have static IP addresses assigned to them".
+> Not meeting this pre-requisite won't prevent the playbook from working but the cluster might will most likely stop working in case one of the server in the architecture changes IP address.
+
+For example:
+
+```yaml
+...
+  repository:
+    vars:
+      cs_storage:
+        type: nfs
+        device: nas.infra.local:/exports/contentstore
+        options: _netdev,noatime,nodiratime
+    hosts:
+      ecm1.infra.local:
+      ecm2.infra.local:
+      ingester.infra.local:
+        cluster_keepoff: true
+...                                                                                                                                                     ```                                                                                                                                                     
```

Comment:
check closing fences not on a new line

Reply:
I'm fixing this and loads of other stuff